lib/openstudio/aws/aws.rb

```diff
@@ -205,19 +212,9 @@ def create_workers(number_of_instances, options = {}, user_id = 'unknown_user')
           # end
 
           @os_aws.launch_workers(options[:image_id], options[:instance_type], number_of_instances, worker_options)
-
-          # Add the worker data to the JSON
-          save_cluster_json @instances_filename
-
-          puts ''
-          puts 'Worker SSH Command:'
-          @os_aws.workers.each do |worker|
-            puts "ssh -i #{@os_aws.private_key_file_name} ubuntu@#{worker.data[:dns]}"
-          end
         end
 
-        puts ''
-        puts 'Waiting for server/worker configurations'
+        logger.info 'Waiting for server/worker configurations'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.